tests/test_annotated_types.py

```diff
@@ -215,7 +215,13 @@ class OptionalUser(LegacyTypedDict, total=False):
     class User(OptionalUser):
         id: int
 
-    with pytest.raises(TypeError, match='^You should use `typing_extensions.TypedDict` instead of `typing.TypedDict`'):
+    with pytest.raises(
+        TypeError,
+        match=(
+            '^You should use `typing_extensions.TypedDict` instead of `typing.TypedDict` '
+            'or upgrade to Python >= 3.9.2'
```

Comment:
you need to update the test as well

Reply:
Ah, good call. Fixed as of https://github.com/samuelcolvin/pydantic/pull/3374/commits/b49afccdf1b92298ca039a93510cca0dbc84fa47.